acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -1,5 +1,9 @@
 test_name "concurrent catalog requests (PUP-2659)"
 
+tag 'audit:low',
+    'audit:integration',  # Testing server
+    'server'
```

Comment:
is that comment in the wrong place?  or can it be an integration test because we're 'testing server'?

i agree the catalog endpoint being able to handle concurrent requests should be tested in its integration tests, and while it is busy, as here.